modules/services/pbgopy.nix

```diff
@@ -5,18 +5,41 @@ with lib;
 let
   cfg = config.services.pbgopy;
   package = pkgs.pbgopy;
+
+  commandLine = concatStringsSep " " ([ "${package}/bin/pbgopy serve" ]
+    ++ optional (cfg.port != 9090) "--port ${toString cfg.port}"
+    ++ optional (cfg.cache.ttl != "24h") "--ttl ${cfg.cache.ttl}"
+    ++ optional (cfg.httpAuth != "") "--basic-auth ${cfg.httpAuth}");
 in {
   meta.maintainers = [ maintainers.ivar ];
 
   options.services.pbgopy = {
     enable = mkEnableOption "pbgopy";
 
+    port = mkOption {
+      type = types.int;
+      default = 9090;
+      example = 8080;
+      description = ''
+        The port to host the pbgopy server on.
+      '';
+    };
+
     cache.ttl = mkOption {
       type = types.str;
       default = "24h";
       example = "10m";
       description = ''
-        The TTL for the cache. Use <literal>"0s"</literal> to disable it.
```

Comment:
Is a cache TTL of 0 second no longer supported?

Reply:
Not sure why i removed this, but no. That should still work perfectly fine, thanks for pointing that out :)